packages/terra-dialog/tests/jest/Dialog.test.jsx

```diff
@@ -4,26 +4,12 @@ import Dialog from '../../src/Dialog';
 describe('Dialog', () => {
   const defaultRender = <Dialog header="Header Content" footer="Footer Content">some body content</Dialog>;
 
-  // Snapshot Tests
+  // Snapshot Test
   it('should render a default component', () => {
     const wrapper = shallow(defaultRender);
     expect(wrapper).toMatchSnapshot();
   });
 
-  // Prop Tests
-  it('should render a header', () => {
-    const container = <Dialog header="Header Content">children required</Dialog>;
-    const wrapper = shallow(container);
-    expect(wrapper).toMatchSnapshot();
-  });
-
-  it('should render header & footer', () => {
```

Comment:
Talked with Neal and Brett last week. Footer is required, so don't need the special test cases with and without.